digitalidentity/service.go

```diff
@@ -145,3 +152,120 @@ func GetShareQr(httpClient requests.HttpClient, sessionID string, clientSdkId, a
 
 	return share, err
 }
+
+// Get Receipt info using the supplied receiptId
+func GetReceipt(httpClient requests.HttpClient, receiptId string, clientSdkId, apiUrl string, key *rsa.PrivateKey) (share ReceiptResponse, err error) {
```

Comment:
Out of curiosity, Is this method public? Integrators should only access the `GetShareReceipt` that does all the fetching and decrypting under the hood.

Reply:
Updated to be private